### PR TITLE
fix: use ImageBackground wrapped in View

### PR DESCRIPTION
**Pull request by craigbilner**
RN 50 deprecated nested children within an `Image`, however, there is an undocumented component called `ImageBackground` which can drop in and provides similar functionality. The existing solution doesn't particularly work for `Android` which doesn't support `position: absolute` very well

### CODE REVIEW

File: packages/image/image.web.js [hunk 1]
```diff
@@ -58,6 +60,14 @@ class TimesImage extends Component {
...
-TimesImage.propTypes = imagePropTypes;
+TimesImage.defaultProps = {
+  style: {}
+};
+
+TimesImage.propTypes = {
+  uri: PropTypes.string.isRequired,
+  aspectRatio: PropTypes.number.isRequired,
+  style: ViewPropTypesStyle
+};
```
Comment (trevorah) [inline packages/image/image.web.js]: do they not share the same propTypes now?

File: packages/image/image.js [hunk 2]
```diff
@@ -30,25 +22,40 @@ class TimesImage extends Component {
...
   render() {
-    const { uri: dirtyUri, aspectRatio, style } = this.props;
+    const { uri: dirtyUri, style, aspectRatio } = this.props;
     const { isLoaded } = this.state;
-    // web handles missing protocols just fine, native doesnt. This evens out support.
+    // web handles missing protocols just fine, native doesn't. This evens out support.
     const uri = addMissingProtocol(dirtyUri);
 
+    const props = {
+      style: [{ width: "100%", height: "100%" }].concat(style),
```
Comment (trevorah) [inline packages/image/image.js]: why not `flex-grow` and `align-items: stretch`?

File: packages/image/__tests__/web/image.web.test.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ it("allows styling", () => {
 
 it("prepends https schema", () => {
   const tree = renderer
-    .create(<Image uri="//example.com/image.jpg" aspectRatio={1 / 1} />)
+    .create(<Image uri="//example.com/image.jpg" aspectRatio={3 / 2} />)
```
Comment (trevorah) [inline packages/image/__tests__/web/image.web.test.js]: why?

File: packages/image/__tests__/image.native.test.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("prepends https schema", () => {
+    const tree = renderer
+      .create(<Image uri="//example.com/image.jpg" aspectRatio={3 / 2} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+};
```
Comment (trevorah) [inline packages/image/__tests__/image.native.test.js]: What platform does this test run with?

File: packages/image/__tests__/web/image.web.test.js (hunk 3 above)
Comment (trevorah) [reply packages/image/__tests__/web/image.web.test.js]: 🗣 easier to test

File: packages/image/__tests__/image.native.test.js (hunk 4 above)
Comment (trevorah) [reply packages/image/__tests__/image.native.test.js]: 🗣 not actually a test, just a func that supplies tests

Comment (craigbilner) [reply packages/image/__tests__/image.native.test.js]: just common test code

File: packages/image/image.js (hunk 2 above)
Comment (craigbilner) [reply packages/image/image.js]: same concept

Comment (trevorah) [reply packages/image/image.js]: 🗣 same difference

**Pull request merged**